.prettierrc.js

```diff
@@ -2,6 +2,7 @@
  * @type {import('prettier').Options}
  */
 module.exports = {
+  plugins: ['prettier-plugin-packagejson'],
```

Comment:
It appears that Prettier no longer autodetects plugins in v3. I couldn't find any mention of this in the [release notes](https://prettier.io/blog/2023/07/05/3.0.0.html), but I found this necessary to add to format `package.json` correctly.